GitUI/CommandsDialogs/SettingsDialog/Pages/AdvancedSettingsPage.cs

```diff
@@ -64,5 +64,12 @@ private void chkAutoNormaliseBranchName_CheckedChanged(object sender, EventArgs
         {
             cboAutoNormaliseSymbol.Enabled = chkAutoNormaliseBranchName.Checked;
         }
+
+        private void chkConsoleEmulator_InfoClicked(object sender, EventArgs e)
```

Comment:
I feel that a more strategic/robust change would be to make a change at the `SettingsCheckBox` level similar how we have the documentation-related configurations in the `GitExtensionsDialog`:
https://github.com/gitextensions/gitextensions/blob/603144eb249e022d0e104fcb30ff79b41e524476/GitUI/GitExtensionsDialog.cs#L31-L47
https://github.com/gitextensions/gitextensions/blob/603144eb249e022d0e104fcb30ff79b41e524476/GitUI/GitExtensionsDialog.cs#L73-L74

This way we won't need to to configure that many handlers all over the place, nor need to hardcode the URLs. We'd just need to hardcode `ManualSectionSubfolder=settings` and `ManualSectionAnchorName={...}` for each control. 
That would be much smaller and more targeted change IMO.

Reply:
This is not just SettingsCheckBox  but also PictureBox that need to be subclassed if changed.
It could be possible to use OsShellUtil.OpenUrlInDefaultBrowser() without a handler too or use Tag without any methods changed.

I prefer to keep this.

Note: The recent change to cursor hand on the infoboxes (already on PictureBox?) more or less require that the change is done (but helpful also before that).

Side note: I considered changing to switch Settings to use GitExtensionsForm. A relative straight forward change, but there were a few display issues.

Reply:
I am going to suggest a variant with a property.

> Note: The recent change to cursor hand on the infoboxes (already on PictureBox?) more or less require that the change is done (but helpful also before that).

Yes. Obviously, I chose one of the very few with an existing link handler for testing, i.e. I did not notice that most others had none yet.

Reply:
What do you think about 08260ecd8408f20fe846fd5b22b097552b570abe?

Reply:
I had done this in parallel....
I had used the same name as in GitExtensionsForm, this is related (except the the page has a default value.

Feel free to change as you prefer. (I have not done or planned PictureBox now and DiffViewer is changed in the Difftastic PR)